az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/HadoopJobUtils.java

```diff
@@ -125,7 +125,7 @@ public static HadoopSecurityManager loadHadoopSecurityManager(Props props, Logge
     HadoopSecurityManager hadoopSecurityManager = null;
 
     try {
-      Method getInstanceMethod = hadoopSecurityManagerClass.getMethod("createInstance", Props.class);
```

Comment:
I believe we need to revert this change in some other places. There are more occurrences of `hadoopSecurityManagerClass.getMethod` in https://github.com/azkaban/azkaban/pull/2313